### PR TITLE
Reusable templates

### PR DESCRIPTION
**Pull request by sudhacheran**
Changes for creating a reusable template for Project Information Input Date in all the reports

### CODE REVIEW

File: src/components/mdx_wrapper.tsx [hunk 1]
```diff
@@ -61,7 +63,7 @@ const MdxWrapper: FC<MdxWrapperProps> = ({Component}) => {
         return (
           <div className="container" id="mdx-container">
             {/* doc will be undefined for the very first render */}
```
Comment (charliepnnl) [inline src/components/mdx_wrapper.tsx:65]: Could you change the comment to `/* metadata and data will be undefined ...`?

File: src/templates/doe_workflow_hpwh.mdx [hunk 2]
```diff
@@ -2,23 +2,7 @@
 
 <Tabs>
   <Tab eventKey="project" style={{paddingTop:"1rem"}} title="Project">
-    The data for this tab should be know prior to the initial site visit.
```
Comment (charliepnnl) [inline src/templates/doe_workflow_hpwh.mdx:5]: I suggest keeping line 5 "The data for this ..." separate from ProjectInfoInputs. I can imagine using a different sentence here for different workflows. For example, I think the building number photo input is included in the first tab in one of the workflows, and that photo can't be taken prior to the initial site visit.

Also, it should be " ... known ...". The "n" is missing.

File: src/templates/doe_workflow_hpwh.mdx [hunk 3]
```diff
@@ -2,23 +2,7 @@
...
-    <StringInput label="Company Address" path="installer.mailing_address" />
-    <StringInput label="Company Phone" path="installer.phone" />
-    <StringInput label="Company Email" path="installer.email" />
-
-    ##### Project Address
-    <StringInput label="Street Address" path="location.street_address" />
-    <StringInput label="City" path="location.city" />
-    <USStateSelect label="State" path="location.state" />
-    <StringInput label="Zip Code" path="location.zip_code" />
+    <ProjectInfoInputs {...props} />
```
Comment (charliepnnl) [inline src/templates/doe_workflow_hpwh.mdx:5]: Nice!!

File: src/templates/reusable/project_info_inputs.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+    The data for this tab should be know prior to the initial site visit.
```
Comment (charliepnnl) [inline src/templates/reusable/project_info_inputs.mdx:2]: See my comment above about leaving this first sentence out of ProjectInfoInputs template.

**Review by charliepnnl — DISMISSED** [review]
Approved with comments that should be addressed.
Once those comments are addressed, please merge this into main.

**Review by charliepnnl — APPROVED** [review]
Looks good

File: src/components/mdx_wrapper.tsx (hunk 1 above)
Comment (sudhacheran) [reply src/components/mdx_wrapper.tsx]: done

File: src/templates/reusable/project_info_inputs.mdx (hunk 4 above)
Comment (sudhacheran) [reply src/templates/reusable/project_info_inputs.mdx]: Done

File: src/templates/doe_workflow_hpwh.mdx (hunk 2 above)
Comment (sudhacheran) [reply src/templates/doe_workflow_hpwh.mdx:5]: Moved it accordingly

**Pull request merged**